src/molecules/Input/Input.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Inputfield, Label } from "../../atoms";
+import { ICustomInputProps } from "../../atoms/InputField";
+import { ICustomLabelProps } from "../../atoms/InputLabel";
+import "./input.scss";
+
+const Input = (
+  props: React.PropsWithChildren<ICustomInputProps & ICustomLabelProps>
+): React.ReactElement => {
+  return (
+    <div>
+      <Label {...props} />
```

Comment:
I see what you're doing here spreading the props into both components but I'm not sure if it's the way that we should be handling it, it just feels like we're barrelling towards disaster. I say we:
1. Expand the props syntax to actually destructure all label-specific props
2. Rest all of the other props into a variable that we can spread into the `Input` like this